src/Microsoft.Bcl.AsyncInterfaces/ref/Microsoft.Bcl.AsyncInterfaces.cs

```diff
@@ -62,6 +62,11 @@ public sealed partial class AsyncIteratorStateMachineAttribute : System.Runtime.
             public System.Runtime.CompilerServices.ConfiguredValueTaskAwaitable<bool> MoveNextAsync() { throw null; }
         }
     }
+    [System.AttributeUsageAttribute(AttributeTargets.Parameter, Inherited = false)]
+    public sealed class EnumeratorCancellationAttribute : System.Attribute
```

Comment:
Don't you need to add the typeforward on the Forwards.cs file as well so that this gets resolved to System.Runtime?

Reply:
Yup. Thanks.